js/data.js

```diff
@@ -1 +1,20 @@
 /* exported data */
+
+var data = {
+  view: 'search-page',
+  results: [],
+  nextResultId: 1,
+  entries: [],
+  nextEntryId: 1
```

Comment:
What's the difference between `nextResultId` & `nextEntryId`

Reply:
This is for a future issue, but I was planning to transfer all search results into their own array on load, then transfer that data to the entries array if a user decides add shows to their watchlist